account/src/main/java/bisq/account/AccountService.java

```diff
@@ -33,19 +33,17 @@
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 
-import javax.annotation.Nullable;
 import java.util.*;
 import java.util.concurrent.CompletableFuture;
 import java.util.stream.Collectors;
 
 @Slf4j
+@Getter
```

Comment:
Minor additional change. All instance members were annotated with a Getter, rather moved it to a class level.